ksonnet-gen/printer/printer.go

```diff
@@ -428,7 +428,13 @@ func (p *printer) print(n interface{}) {
 			return
 		case ast.StringSingle, ast.StringDouble:
 			useSingle := (qm != quoteModeDouble)
-			quoted := stringQuote(t.Value, useSingle)
+
+			// Unescape newlines and tabs. The jsonnet parser will escape
+			// these during parsing.
+			val := strings.Replace(t.Value, "\\n", "\n", -1)
+			val = strings.Replace(val, "\\t", "\t", -1)
```

Comment:
This leaves out additional characters that can be double-escaped by the current implementation, specifically: `\/bfr`. This can be enhanced in a future fix.